src/utils/utility.ts

```diff
@@ -0,0 +1,83 @@
+import useAuthorize from 'Api/hooks/useAuthorize';
+import { getCurrentDomain } from './websocket/config';
+
+type TObjParams<T extends number> = {
+    [K in `acct${T}`]: string;
+} & {
+    [K in `cur${T}`]: string;
+} & {
+    [K in `token${T}`]: string;
+};
+
+type TModifiedAccountsList = NonNullable<ReturnType<typeof useAuthorize>['data']['account_list']>[number] & {
+    token?: string;
+};
+
+type TClientObject = Record<string, TModifiedAccountsList>;
+
+export const storeClientAccounts = (
+    obj_params: TObjParams<number>,
+    account_list: ReturnType<typeof useAuthorize>['data']['account_list']
+) => {
+    const map_names: { [key: string]: string } = {
+        country: 'residence',
+        landing_company_name: 'landing_company_shortcode',
+    };
+    const client_object = {} as TClientObject;
+    let active_loginid = '';
+
+    account_list?.forEach(account => {
+        Object.keys(account).forEach(param => {
+            if (param === 'loginid') {
+                if (!active_loginid && !account.is_disabled) {
+                    if (!account.is_virtual) {
+                        active_loginid = account[param] ?? '';
+                    } else if (account.is_virtual) {
+                        // TODO: [only_virtual] remove this to stop logging non-SVG clients into virtual
+                        active_loginid = account[param] ?? '';
+                    }
```

Comment:
```suggestion
                   active_loginid = account[param] ?? '';
```
can we change it for now? 
I see that there is a TODO for the `else if` block so we can add more logics but for now I think we can remove it. what do u think @henry-deriv ?

Reply:
@shayan-deriv i changed it. Take a look!

Reply:
thanks man. LGTM